app/Filament/Resources/DatabaseHostResource.php

```diff
@@ -10,7 +10,7 @@ class DatabaseHostResource extends Resource
 {
     protected static ?string $model = DatabaseHost::class;
 
-    protected static ?string $label = 'Databases';
+    protected static ?string $label = 'Database Host';
```

Comment:
I'm not saying this is wrong, but this is what Pterodactyl had before. Just food for thought.

Reply:
I think for clarification, Database Hosts is correct. It's referred to that in discord when people talk and on the docs